### PR TITLE
feat(docker): Add support for multi-platform build

### LINKED ISSUE #559: feat(): Multiarch support
Hey! I was looking for an option to cross-compile some program, long story short: cross compilation is not supported.

And I've discovered that docker support `--platform` switch.

Setup required:

 - `# apt install -y qemu binfmt-support qemu-user-static`
 - `# docker run --rm --privileged multiarch/qemu-user-static:register`

And while this solution is nowhere close to being fast, it works, it runs `arm` images on `amd64` host.

```bash
root@oldpc ~# docker run --rm -ti --platform linux/arm/v7 debian:latest uname -m
armv7l
```

Which mean that you could do a *native* build of arm64 binaries on amd64 host, in case when cross compilation is not an option. Or you could test if some edge-case happen on arm hosts.

What do you think about bringing support of `--platform` switch to the Abstruse CI? In my opinion it should be configurable in the `matrix` in `.abstruse.yml`, like this:

```yml
platform: host // Special platform that just take `uname -m` as platform
matrix:
  - platform: arm64
  - platform: arm/v7
```

### PR DESCRIPTION
**Pull request by MrCyjaneK**
This pull request close #559

Since docker already support multiarch by using qemu, it would be cool to have abstruse support it also,

Commands required to setup:
```bash
(HOST)# apt install -y qemu binfmt-support qemu-user-static
(HOST)# docker run --rm --privileged multiarch/qemu-user-static:register
```

Currently there is no gui for setting platforms (and it is not likely that I'll do that.. I have no idea how frontend works), so in database you need to update `platforms` row with something like: `linux/amd64;linux/arm64`

After that trigger a build and you will see this:

| linux/arm64 | linux/amd64 |
| ---- | ---- |
| image | image |

### CODE REVIEW

File: server/core/repo.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type (
 		Provider      Provider      `json:"-"`
 		EnvVariables  []EnvVariable `json:"-"`
 		Mounts        []*Mount      `json:"mounts"`
+		Platforms     string        `json:"platforms"`
```
Comment (belsakn) [inline server/core/repo.go:38]: This should be an array?
```suggestion
		Platforms     []string        `json:"platforms"`
```

File: server/store/build/build.go [hunk 2]
```diff
@@ -318,31 +318,33 @@ func (s buildStore) TriggerBuild(opts core.TriggerBuildOpts) ([]*core.Job, error
...
-			BuildID:  build.ID,
-			Cache:    strings.Join(j.Cache, ","),
-		}
-		if err := s.jobs.Create(job); err != nil {
-			return nil, err
-		}
-		job, err = s.jobs.Find(job.ID)
-		if err != nil {
-			return nil, err
+	for platform := range strings.Split(repo.Platforms, ";") {
```
Comment (belsakn) [inline server/store/build/build.go:321]:
```suggestion
	for _,platform := range repo.Platforms {
```

File: server/core/repo.go (hunk 1 above)
Comment (MrCyjaneK) [reply server/core/repo.go:38]: Yeah! That will make things easier

**Comment by MrCyjaneK** [comment]
@belsakn `panic: invalid sql type  (slice) for sqlite3` I got this error when I tried to use `[]string`
